.github/workflows/latest_import.yml

```diff
@@ -0,0 +1,129 @@
+name: Weekly late importers
+
+on:
+  schedule:
+    - cron: '0 4 * * 1'  # Run at 04:00 UTC every Monday
+  workflow_dispatch:
+
+jobs:
+  late-importers:
+    name: Build
+    runs-on: pr
+    steps:
+    - name: cleanup
+      run: |
+        find . -name . -o -prune -exec rm -rf -- {} +
+        # Delete all but the 5 most recent toolchains.
+        # Make sure to delete both the `~/.elan/toolchains/X` directory and the `~/.elan/update-hashes/X` file.
+        # Skip symbolic links (`-type d`), the current directory (`! -name .`), and `nightly` and `stable`.
+        if cd ~/.elan/toolchains && find . -maxdepth 1 -type d ! -name . -print0 | xargs -0 ls -1td | grep -v 'nightly$' | grep -v 'stable$' | tail -n +6 | xargs -I {} sh -c 'echo {} && rm -rf "{}" && rm "../update-hashes/{}"'; then
+            : # Do nothing on success
+        else
+            : # Do nothing on failure, but suppress errors
+        fi
+
+    # The Hoskinson runners may not have jq installed, so do that now.
+    - name: 'Setup jq'
+      uses: dcarbone/install-jq-action@v2.1.0
+
```

Comment:
I think but I'm not 100% sure this can be deleted:
```suggestion
```

I think in the main CI workflow jq is only used in `lean-pr-testing-comments.sh`. Note to self: if true, it would be good to document that.

Reply:
I am going to test the removal in #18636, once it is done doing its "full run".

Reply:
I think that the test was successful: this step is not needed if you just want to build mathlib.